### ISSUE TITLE
Unconscious Animations was in version 3.5.0 but in version 3.6.2 this not works.

### ISSUE THREAD

**Issue by mickeymen** [acemod/ACE3#4182]
**Arma 3 Version:** `1.62` (stable)
**CBA Version:** `3.0.0.160713` (stable)
**ACE3 Version:** `3.6.2` (stable)

**Mods:**
- `@CBA_A3`
- `@ace`
- `@JSRS4 (!only sound mod)`

**Description:**
Hi! recently I observed accidentally.
In version 3.5.0, the ACE team tried to create a good animation, when a soldier lost consciousness using ragdol animation. 
Example - https://www.youtube.com/watch?v=dukLGkq2WQo&feature=youtu.be
This is a good news! However, in version 3.6.2 it is not is present. This is strange moment. 
Player see again, not good animation of unconscious units. 
Example - https://www.youtube.com/watch?v=ku9y4-Rbmyw

**I would like to know - This is a mistake of version 3.6.2, or the intention of developers?**

Animations in version 3.5.0, have been improved and it shows progress, but this animation does not always works good.  I think, that only animation at number 3 (my videos proposed below), looks completely fine, except that the rotation speed should be slightly smaller than. Always this animations looked differently. Maybe this is the reason for which it was not included in version 3.6? ...I have only guesses.  I realy hope, that the this work will be completed, and we can see in the next versions the good animation for unconscious units. It will be great progress for ACE3.

Examples of animations from version 3.5.0. It's always worked differently and ambiguously:

1. When a soldier falls on his back, it is not turning animation to the prone position.
https://www.youtube.com/watch?v=dukLGkq2WQo&feature=youtu.be

2. If a soldier falls face down, then he has excellent animation of the rotation to his back. At this point, he twitches in pain. Cool! But however, again there is no animation rotation from his back to the prone position:  https://www.youtube.com/watch?v=SE0jhzFNkBk&feature=youtu.be

3. In the third example, everything works great(if rotation speed will be reduced)! So, it is possible in the ACE3! All should work thus: https://www.youtube.com/watch?v=bANhWFW-sN8&feature=youtu.be

ACE, team, In my opinion, if you correct the following items, then the unconscious will look great!
And yet, also one the item. The soldier too quickly recovers to consciousness. As seems to me, unit must be more long in a state of ragdoll. For example, the status of ragdoll(unconscious), should be within a random time, between from 3-4-5 to 10-15-18 seconds.
Thanks for attention and good works!

**Steps to reproduce:**
- Compare versions 3.5.0 and 3.6.2

**Where did the issue occur?**
- Editor (Singleplayer) 

**Placed Modules:**
- None

**RPT log file:**
- Not needed

**Comment by esteldunedain** [comment]
> In version 3.5.0, the ACE team tried to create a good animation, when a soldier lost consciousness using ragdoll animation.

No, those are not ragdoll animations. We never introduced ragdolls in ACE and we won't until BIS introduces proper a proper API.

Nothing was changed regarding animation between 3.5 and 3.6.2. My guess is you are not seeing any animation on that video because you had very few frames per seconds. I just tested this and it works as in your videos, although the animations are not always the same.

**Comment by commy2** [comment]
Nope. There are no animations. BI has some for their revive system which was interfering with parts of medical. 3.5. was broken after APEX until we fixed some things.

**Comment by PabstMirror** [comment]
@mickeymen - if you want to use the 1.62 unconc animations, feel free to add this code to a mission's init.sqf and see how it looks:

```
["ace_unconscious", {
    params ["_unit", "_state"];
    if (local _unit) then {
        diag_log text format ["setting BIS unconc %1", _this];
        _unit setUnconscious _state;
    };
}] call CBA_fnc_addEventHandler;
```

**Comment by mickeymen** [comment]
Guys. My English is not best, but this is not language problem! I made especially for you 3 video example, but As seems to me, we can not understand each other, again and again!

> Nope. There are no animations. BI has some for their revive system which was interfering with parts of medical. 3.5. was broken after APEX until we fixed some things.

@commy2, Please reed attentenly.
I showed to you three videos examples, with new animations, which exist only, if I use version ACE3 -3.5.0! Please Note! ONLY, if I use version ACE3 -3.5.0! As you can see, in the this examples, injured soldier falls like a rag doll (Note! this is the first animation, that does not exist in version 3.6.2. Rag doll animation not exist in 3.6.2!!! ) and after this fall, this soldier has also a wounded  animation. To be more precise - He writhes in pain and sometimes, even turns over onto his stomach and then gets to his feet. (Again Note! This is the second animation, which does not exist in version 3.6.2. NOT EXIST!) This test, was created with version ACE3 - 3.5.0. and with ARMA3 version 1.62. Note! v 1.62 of Arma3 - this is already APEX version!  So, If I use the ACE version 3.6.2. then these animations would not exist!  They do not exist in version 3.6.2! But you continue say me, hat there is no difference between 3.5.0 and 3.6.2. 

> @mickeymen - if you want to use the 1.62 unconc animations, feel free to add this code to a mission's init.sqf

@PabstMirror You do not understand me. I do not want to use 1.62 unconc animations. Сonversely  I want to use animations from 3.5.0.  When I use ONLY this version, I see ragdoll animation +wounds animations which I want see in the ACE3.

**Comment by jonpas** [comment]
For the last time, nothing was changed on our side. What I think you are seeing was a side effect from bugs that appeared with Arma 3 1.62, which got fixed with ACE3 3.6.0.

And please be civil, if you can't be don't post here, we try our best to help you just to get yelled at and accused of not reading? Thanks, I don't want to help such a man.
